modules/identify-deployments-with-most-severe-policy-violations.adoc

```diff
@@ -1,9 +1,10 @@
+//This widget no longer seems to exist
```

Comment:
This is correct, the widget was removed in a release before 4.5

Reply:
OK, I will remove this, thank you for confirming!